app/src/main/java/com/hover/stax/bonus/BonusViewModel.kt

```diff
@@ -17,7 +18,7 @@ import kotlinx.coroutines.flow.update
 import kotlinx.coroutines.launch
 import timber.log.Timber
 
-class BonusViewModel(val repo: BonusRepo, private val channelRepo: ChannelRepo) : ViewModel() {
+class BonusViewModel(val repo: BonusRepo, private val channelRepo: ChannelRepo,private val presentSimUseCase: GetPresentSimUseCase) : ViewModel() {
```

Comment:
This class will eventually be removed in favor of the `GetBonusUseCase`. Make the changes there as well. Eventually, fetch bonus usages will be directed to the usecase.

Reply:
Code updated.